.changeset/swift-nails-notice.md

```diff
@@ -0,0 +1,13 @@
+---
+"@rhds/elements": minor
+---
+
+`<rh-navigation-secondary>`
```

Comment:
```suggestion
`<rh-navigation-secondary>`:
```

Reply:
Also, probably worth adding a changeset for removing RTI from accordion as well.

Reply:
Oh good catch!